lib/pbench/test/unit/agent/test_tool_group.py

```diff
@@ -123,7 +125,9 @@ def mock_read_text(self: Path):
             """Mocked directory check"""
             raise FileNotFoundError("Mock Path.resolve()")
 
-        monkeypatch.setattr(ToolGroup, "verify_tool_group", self.mock_verify_tool_group)
+        monkeypatch.setattr(
+            ToolGroup, "verify_tool_group", staticmethod(self.mock_verify_tool_group)
+        )
```

Comment:
When we simply remove the `staticmethod()` constructor, this fails.  If we replace it with `Test_ToolGroup.mock_verify_tool_group` instead of `self.mock_verify_tool_group` it also fails.